lld/ELF/LinkerScript.h

```diff
@@ -195,6 +195,7 @@ class SectionPattern {
 
 class InputSectionDescription : public SectionCommand {
   SingleStringMatcher filePat;
+  SmallString<0> implicitArchiveWildcardPat;
```

Comment:
nit: move before since the lifetime of the SmallString should be larger than `filePat`